### ISSUE TITLE
Übersicht der "Icons added"

### ISSUE THREAD

**Issue by rtwl21** [inventwo/ioBroker.vis-icontwo#529]
Gibt es irgendwo eine Übersicht welche Icons hinzugefügt wurden?
ich hab jetzt schon öfters den Adapter aktualisiert, konnte aber nie auf einfache Art herausfinden welche Icons neu hinzu gekommen sind.
eventuell als extra Übersicht nach Datum der Aktualisierung auf eurer Website abbilden?

**Comment by skvarel** [comment]
Hallo. Im Change-Log steht eigentlich immer, was neu hinzugekommen ist. Ich überlege mir mal etwas für die Zukunft.

**Comment by rtwl21** [comment]
Ja, stimmt. In der ReadMe beim Changelog hab ich noch nicht geschaut. Wäre nur eine feine Sache wenn man die neuen Übersichtlich irgendwo sehen kann - hat aber keine Prio. Soll nur als Anregung/Idee dienen.
